lib/controllers/v1/observations_controller.js

```diff
@@ -107,13 +107,32 @@ ObservationsController.searchCacheWrapper = async req => (
 
 ObservationsController.search = async req => {
   if ( req.query.return_bounds === "true" ) {
-    req.query.aggs = {
-      bbox: {
-        geo_bounds: {
-          field: "location"
-        }
+    // If we've been asked to return the bounds but also to return obs in a
+    // place, assume that the bounds are the bounding box of the place and not
+    // the bounding box that contains the observations in that place. This
+    // changes the meaning of return_bounds, but will probably satisfy most
+    // cases and might be more performative
+    if ( req.query.place_id ) {
```

Comment:
we might want to check req.inat.place here instead of the param. We'll fetch the place for the place_id query param in `InaturalistAPI.lookupPlaceMiddleware`, but only if there is a single value for place_id. As currently implemented in this branch, if I send a comma-delimited list of place_ids this will fail on the next line since it'll search on the whole string with commas. By using req.inat.place we'll already have confirmed there is only one place_id and the instance will already be fetched. We'll just need to add `bounding_box_geojson` as a default return field for Place.findByID which seems fine